### ISSUE TITLE
Gem doesn't work with Turbolinks

### ISSUE THREAD

**Issue by jleecbd** [ncri/nested_form_fields#82]
Re: the previous issue I submitted.  With Turbolinks present, clicking on add_nested link results in two sets of fields being created.  I tried moving the load order around on the js, but that had no effect.

**Comment by ncri** [comment]
The gem works fine with turbolinks for us. It sounds like you have attached the event handlers twice? I remember someone had that issue here.

**Comment by jleecbd** [comment]
The problem I had seen was where the nested fields js was loaded twice in a
page.  I went through all of the loaded scripts on the page and only saw it
once.  Removing turbolinks fixed the problem, but is obviously not the most
desirable solution.  I'll take another deeper look and see if it is being
called anywhere else that could produce that phenomenon.

**Comment by crm114** [comment]
@jleecbd load "//=require nested_form_fields" *before* turbolinks on your application.js file

**Comment by jondkinney** [comment]
Agreed @jleecbd. This is the default setup I use for `application.js` now:

```js
// This is a manifest file that'll be compiled into application.js, which will include all the files
// listed below.
//
// Any JavaScript/Coffee file within this directory, lib/assets/javascripts, vendor/assets/javascripts,
// or any plugin's vendor/assets/javascripts directory can be referenced here using a relative path.
//
// It's not advisable to add code directly here, but if you do, it'll appear at the bottom of the
// compiled file.
//
// Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
// about supported directives.
//
//= require jquery
//= require jquery.turbolinks
//= require jquery_ujs
//
// ... your other scripts here ...
//
//= require turbolinks
```

Note that you need to include the jquery turbolinks gem in your gemfile to have this work. The above setup is from their readme https://github.com/kossnocorp/jquery.turbolinks

**Comment by ncri** [comment]
Thanks for those tips
